### ISSUE TITLE
Problems with compiling

### ISSUE THREAD

**Issue by GhostBnuuy** [ThatRozebudDude/FPS-Plus-Public#56]
I tried to compile FPS Plus and I got this error
```C:/HaxeToolkit/haxe/lib/flixel/5,2,1/flixel/graphics/tile/FlxDrawBaseItem.hx:9: characters 8-32 : Type not found : openfl.display.Tilesheet```

I've tried downgrading flixel but i keep getting the same error, am I doing something wrong here? These are the libraries I'm using:

![image](https://github.com/ThatRozebudDude/FPS-Plus-Public/assets/84008574/777ae8cd-3740-4406-baad-d9bcdc5a7fe8)

**Comment by ThatRozebudDude** [comment]
This appears to be a similar issue to [this](https://community.openfl.org/t/hx-characters-1-8-build-failure/13007/13). Take a look and see if it helps.

**Comment by GhostBnuuy** [comment]
Yeah it's not working, I added `<haxedef name="FLX_DRAW_QUADS" />` in Project.xml but I get this error 

`C:/HaxeToolkit/haxe/lib/flixel/5,2,2/flixel/system/macros/FlxDefines.hx:121: characters 59-63 : FLX_DRAW_QUADS can only be defined by flixel.`

**Comment by SanicBTW** [comment]
That means that only Flixel can change that FlxDefine and is not part of the user defines

**Comment by SanicBTW** [comment]
You can override https://github.com/HaxeFlixel/flixel/blob/dev/flixel/system/macros/FlxDefines.hx and in user defines add FLX_DRAW_QUADS so you can modify it but could lead to unwanted behaviour

```
private enum UserDefines
{
	FLX_NO_MOUSE_ADVANCED;
	FLX_NO_GAMEPAD;
	FLX_NO_NATIVE_CURSOR;
	FLX_NO_MOUSE;
	FLX_NO_TOUCH;
	FLX_NO_KEYBOARD;
	FLX_NO_SOUND_SYSTEM;
	FLX_NO_SOUND_TRAY;
	FLX_NO_FOCUS_LOST_SCREEN;
	FLX_NO_DEBUG;
	FLX_RECORD;
	FLX_UNIT_TEST;
	/* additional rendering define */
	FLX_RENDER_TRIANGLE;
	FLX_DRAW_QUADS;
	/* Uses flixel 4.0 legacy collision */
	FLX_4_LEGACY_COLLISION;
	/* Simplifies FlxPoint but can increase GC frequency */
	FLX_NO_POINT_POOL;
	FLX_NO_PITCH;
	FLX_NO_SAVE;
}
```
